client/components/Main/ListPanel.tsx

```diff
@@ -19,7 +19,8 @@ import {onEventCapture} from '../../utils';
 
 import {ListGroup} from '.';
 import {PanelInfo} from '../UI';
-import {Item, Column, Group} from '../UI/List';
+import {Column, Group} from '../UI/List';
+import {IPlanningExtensionConfigurationOptions} from '../../planning-extension/src/extension_configuration_options';
```

Comment:
You shouldn't be importing from an extension directly. See if it works to use the bridge.

Reply:
ok then